src/zoomabletreeview.cpp

```diff
@@ -13,7 +13,9 @@ ZoomableTreeView::ZoomableTreeView(QWidget *parent)
     :QTreeView(parent)
 {
     QFont fixedFont = QFontDatabase::systemFont(QFontDatabase::FixedFont);
-    fixedFont.setPointSizeF(sm_savedFontPointSize);
+    if (sm_savedFontPointSize > 0) {
```

Comment:
In what scenarios can this saved font size be == 0? I'm not sure that you can make it be 0 using the UI (just zooming out).

Reply:
If it was never changed it keeps its default value from line 10, i.e. 0

Reply:
I think Alvin's question can be rephrased as when would sm_savedFontPointSize be ever set to <= 0?  If it's possible, then we might want to fix it at the source too. It looks like the var should only be set to between MIN and MAX font size or system default font size.  Anyway, it does not hurt to add the check.